examples/usbboot.ts

```diff
@@ -13,19 +13,61 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { argv } from 'process';
+
+/**
+ * Unlocking with secureboot
+ *
+ * If you need to unlock an cm4 with secureboot, you need to provide a signed boot-image.
+ * Pass the bootImageFolder flag with the path to the folder containing the signed boot-image
+ */
+
 import ProgressBar = require('progress');
 
 import { scanner, sourceDestination } from '../lib/';
 
 import { pipeSourceToDestinationsWithProgressBar } from './utils';
 
+// Parse command line arguments
+const args = process.argv.slice(2); // removes 'node' and the script name from the args
+const flags: any = {};
+
+args.forEach((arg: string, index: number) => {
+	// Check if the argument is a flag in the format --flag=value
+	if (arg.startsWith('--')) {
+		const key: string = arg.substring(2);
+		const value: string = args[index + 1];
+		flags[key] = value;
+	}
+});
+
+if (!flags.source) {
+	console.log("No source has been provided, won't try to flash anything");
+}
+
+if (flags.bootImageFolder !== '') {
+	console.log(`Using external folder ${flags['bootImageFolder']}`);
+}
+
+if (flags.help) {
+	console.log(
+		'Usage: ts-node usbboot.js --bootImageFolder <bootImageFolder> --source <image>',
```

Comment:
Is this example file meant to used directly?

Reply:
For demo purposes mostly 
I do use it (also multi-destination) as a poor man CLI

Reply:
If your concern is that would become de-facto production, it shouldn't. We still need a proper place to hook this up (the CLI, a new etcherCLI, a proper script in this repo or somewhere else)